client/test-data/bazaar/EdisonPaula/businesses/kücheedison/products/EdisonKaffee.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "Kaffee",
+	"description": "Koestlicher Kaffe",
+	"logo": "QmZzkgNe6B6M9Y3UeGgugwEB56v5qBm35bcPTPnZFNtY7d"
+}
```

Comment:
please add itemCondition: None (or new`?)

Reply:
what about Identifier? 
and category? (again food?)